CounterStrike/src/dev/danablend/counterstrike/runnables/PlayerUpdater.java

```diff
@@ -62,82 +68,41 @@ public void run() {
 
             if (player.getGameMode().equals(GameMode.SPECTATOR)) {
                 if (player.getSpectatorTarget() != null) {
-                    PacketUtils.sendActionBar(player, Color.YELLOW + "Spectating: " + ChatColor.GREEN + player.getSpectatorTarget().getName(), 1);
+                    PacketUtils.sendActionBar(player, Color.YELLOW + "Spectating: " + ChatColor.GREEN + player.getSpectatorTarget().getName());
                 }
             }
         }
 
     }
 
 
-    public void setScoreBoard(Player player) {
+    public void setScoreBoard(CSPlayer csplayer) {
+        Player player = csplayer.getPlayer();
 
-        CSPlayer csplayer = CounterStrike.i.getCSPlayer(player, false, null);
+        if (!player.isOnline()) return;
 
         //inits player colour
         player.setPlayerListName(ChatColor.valueOf(csplayer.getColour()) + player.getName());
 
-        Scoreboard board = Bukkit.getScoreboardManager().getNewScoreboard();
+        FastBoard board = new FastBoard(player);
+        board.updateTitle(ChatColor.BOLD + "----Miner Strike v" + CounterStrike.i.getDescription().getVersion() + "----");

```

Comment:
This probably should be configurable

Reply:
Scoreboard are no longer supported in Folia, so just changed everything to a way of doing the same in all types of servers

Reply:
I mean the name of the title `----Miner Strike v`

Reply:
That should be our "brand" name, we can do a little brain storm a get a more definitive one.
If we make it configurable we will start seeing sings like DJMaxKiller Plugin version xpto :P